ui/apps/platform/src/Containers/SystemConfig/Form/SystemConfigForm.tsx

```diff
@@ -177,7 +175,7 @@ const SystemConfigForm = ({
                     const isTelemetryEnabledCurr = data.publicConfig?.telemetry?.enabled;
                     const isTelemetryEnabledPrev = publicConfig.telemetry?.enabled;
 
-                    if (isTelemetryEnabledCurr && isTelemetryConfigured) {
+                    if (isTelemetryEnabledCurr && isTelemetryConfigured && telemetryConfig) {
```

Comment:
`telemetryConfig` can be `undefined`, but the redux implementation had it implicitly typed as `any`.